src/util/logging.cpp

```diff
@@ -158,6 +164,13 @@ inline void writeToFile(
     // Writing to a closed QFile could cause an infinite recursive loop
     // by logging to qWarning!
     if (s_logfile.isOpen()) {
+        if (s_logMaxFileSize >= 0 && s_logfile.pos() >= s_logMaxFileSize) {
+            formattedMessage =
+                    "Maximum log file size reached. It can be adjusted via: "
+                    "--log-max-file-size <bytes>";
+            s_logMaxFileSizeReached = true;
+            flush = true;
```

Comment:
shouldn't this be false?

Reply:
This is the last line written to the file and flushing makes sure that it is also immediately written to the HDD.